gnovm/cmd/gno/internal/pkgdownload/pkgfetcher.go

```diff
@@ -0,0 +1,10 @@
+package pkgdownload
+
+type PackageFetcher interface {
+	FetchPackage(pkgPath string) ([]PackageFile, error)
```

Comment:
Looks redundant with a MemPackage :P

Reply:
I replaced `PackageFile` with the existing `gnovm.MemFile` here d87b8768c5b43e335603dcc5ccaca9b76d56cc0e 

I don't think returing a `gnovm.MemPackage` is a good idea since it has a `Name` field that requires parsing the `.gno` files to populate properly. We could return a partial `MemPackage` but I don't understand the advantage vs the list of files, I think it could create confusion actually.